source/extensions/tracers/common/ot/opentracing_driver_impl.h

```diff
@@ -46,6 +46,7 @@ class OpenTracingSpan : public Tracing::Span, Logger::Loggable<Logger::Id::traci
 
   // TODO: This method is unimplemented for OpenTracing.
   std::string getTraceIdAsHex() const override { return EMPTY_STRING; };
+  void setStreamInfo(const StreamInfo::StreamInfo&) override{};
```

Comment:
Unnecessary semicolon. And looks like all the tracers copied the `getTraceIdAsHex()` which also has a unnecessary semicolon.